medical_patient_disease_allergy/demo/medical_patient_demo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="res_partner_patient_5" model="res.partner">
+        <field name="name">Michael Marine</field>
+        <field name="city">Las Vegas</field>
+        <field name="state_id" ref="base.state_us_23" />
+        <field name="country_id" ref="base.us" />
+        <field name="zip">89521</field>
+        <field name="street">123 Test 2 Drive</field>
+        <field name="email">michael@example.com</field>
+        <field name="phone">+1 (702) 3542-2447</field>
```

Comment:
I don't really like the idea of having a phone number that isn't formatted correctly for the country. Maybe we could stick to +1 (702) 555-xxxx for demo data?